src/main/resources/sessionLinksRecoveryEmailTemplate-feedbackSessionAccessLinks.html

```diff
@@ -0,0 +1,26 @@
+<p>Hello ${userName},</p>
+
+<p>
+    You are receiving this email because a request to resend TEAMMATES links related to this email address <strong>${recoveryEmail}</strong> was made on <a href="${teammateHomePageLink}">TEAMMATES</a>.
+</p>
+
+<p>
+    Below are the sessions that are associated with the email <strong>${recoveryEmail}</strong> within the past 180 days:
+</p>
+
+${linksFragment}
+
+<p>
+    If the course you expected to see is missing, it may be associated with a different email address. You can try <a href=${sessionsRecoveryLink}>
```

Comment:
>To recover older session links, they need to contact us. They might not be able to register if they don't have a join link. We can mention that.

Here as well. Since we r explaining y he might not see his expected outcome, we shld tell him all the reasons: 1. a wrong email address 2. session too old. To solve these two problems, 1. check with his instructor 2. contact teammates support team. Hope this makes more sense.

Reply:
@amrut-prabhu  Hi, would you mind help me write proper English words here? My written English is quite bad...